### ISSUE TITLE
Sync property with inhabitants

### ISSUE THREAD

**Issue by jlsjonas** [TWiStErRob/TWiStErRob#53]
Allow linking a property with other inhabitants, ideally allowing an item owner (including "common!") which should prevent edits (except if common//to temporary storage #47 , maybe alerting the owner if not common).
Personal and common items should be listed above "other's" items
Item UUID's would be recommended to prevent sync issues.

**Comment by TWiStErRob** [comment]
Wow, that's an Epic :)
The problem is that the app doesn't have Internet and I wasn't planning on including that permission.

**Comment by jlsjonas** [comment]
I've seen apps sync up without internet permission (don't ask how 😂 )
you could keep it a 6.0+ feature? asking for the permission only when you need it ;)
or using add-on app like some have done (to avoid asking for internet on main app)

**Comment by TWiStErRob** [comment]
huh, plugin sounds like a good idea. But this is far future, because the current content provider is not ready for any of this.

**Comment by jlsjonas** [comment]
Obviously, but rather have the details (somewhat) worked out before messing things up later on 😄

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/157d17b04130ff51

**Comment by jlsjonas** [comment]
uhm... sure you got the right link there? 😏 @TWiStErRob

**Comment by TWiStErRob** [comment]
Don't mind these, I just note some feedback mails for myself. (For more details/alerting when done) You won't have the conversation in your mailbox ;)

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/1597a14687e2fc08

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/159a4148bd2f1e52

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15a5258c751a5fb0

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15a615c998fdb1ef

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/?zx=hpc9wquw0915#inbox/15aad969226f6699

**Comment by TWiStErRob** [comment]
https://play.google.com/apps/publish/?account=7995455198986011414#ReviewDetailsPlace:p=net.twisterrob.inventory&reviewid=gp:AOqpTOFfY9COXpRwAaNFGAdq6eyFu7z2YDtJnPrgWxK4IiTzDG54DQucog4JUoJjU7MCgULbrxaT2_9hcCgP

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15b0a99a4259725c

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15cb739c3e6b634d?compose=15c6e4b28d37c50b

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15cd462861cd6621

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15d4b4278670b060

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15d74de6489a838d

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15d804238e077195

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15ef45428d1131dd

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15f458e3fa4b1fc8

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/15f92225e9ccded7

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/161a9386d8c85f16

**Comment by TWiStErRob** [comment]
Many-many use cases: https://mail.google.com/mail/u/0/#inbox/1621a0aff5fdccca

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/16382c2ed0eb49c9

**Comment by TWiStErRob** [comment]
> Is it possible to be able to sync with other devices
https://mail.google.com/mail/u/0/#search/inventory/16229c341fae065f

**Comment by TWiStErRob** [comment]
> If there is a catastrophic event, I have many things to grab before my phone. I also have multiple devices I would like to use the application on. Sharing the data between the devices is cumbersome I  the  current form.
https://mail.google.com/mail/u/0/#inbox/FMfcgxvzKkmXlsdhpdtDkshqVkghrWXs

**Comment by TWiStErRob** [comment]
for backup: https://mail.google.com/mail/u/0/#inbox/FMfcgxvzKbRHnWsGdlHsFXdNlNsXLNNv

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/FMfcgxvzKtXnwvcGHmgjBspQQPDZQPdD

**Comment by TWiStErRob** [comment]
device sync: https://mail.google.com/mail/u/0/#inbox/FMfcgxvzLNZmFbwQFPkWFzpwZwCBNzDr

**Comment by TWiStErRob** [comment]
> It is a little difficult to have multiple people packing and maintaining synchronization.
https://mail.google.com/mail/u/0/#inbox/FMfcgxvzLXJjHSgKhwpHmjBLmDcfnzMQ

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/#inbox/FMfcgxwBVWQNkKWPwXxSrNWQTNpsKmxK

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/?zx=7kp0af6y11dv#inbox/FMfcgxwDrvCQvFnbcswvktgZwHqzkVKp

**Comment by TWiStErRob** [comment]
https://mail.google.com/mail/u/0/?zx=fub1bmh7xeys#inbox/FMfcgzGqQSNqZkCjsMpgDvFDzDRsDgzq

**Comment by TWiStErRob** [comment]
Multiple devices: https://play.google.com/console/u/0/developers/7995455198986011414/app/4974852622245161228/user-feedback/review-details?reviewId=79443d2a-a9e1-44df-871d-cc7d610b4659&corpus=PUBLIC_REVIEWS

**Comment by TWiStErRob** [comment]
A rough workaround implementation could be achieved by #56

**Comment by TWiStErRob** [comment]
> Sharing. I noticed there is a share/export function, where you can send someone an item with its description. This is not bad, but it's not what I was thinking of (.. and by the way: I believe it might be a privacy issue, that the share function does not only show the item but also the "whole path" of the item, thus, the exact position in which container that item is in.. this is probably unwanted if you want to share an item in a WhatsApp group, which you want to sell). What I was actually thinking of was something like shared folders, a little bit similar to Google Drive. You would select certain rooms or containers and share them with another person. This person would then be able to look inside this container as well. And, if permissions are set, he/she could also edit the content. This idea of sharing however, would most likely need some kind of account system and cloud storage for user inventories. Using users Google Drive accounts might work, but might not be the best solution.
> -- https://mail.google.com/mail/u/0/#inbox/FMfcgxmTpCHlHzSXRStfqfxmCMQznZMP / https://mail.google.com/mail/u/0/#inbox/15c276a712283edf

**Comment by TWiStErRob** [comment]
> Is there anyway to sync with google drive or sheets so that I can see the invetory across multiple devices?
> -- WhatsApp 44375
